client/src/enum/api/task-api-path.ts

```diff
@@ -1,3 +1,4 @@
 export const TaskApiPath = {
 	SEARCH: 'tasks/search',
+	ROOT: '/tasks/'
```

Comment:
Why is the search without a slash at the beginning of the word?
And the root has a slash at the beginning of the word